app/src/main/res/values/strings.xml

```diff
@@ -253,6 +253,14 @@
     <string name="preference_tts_message_format_combined">Reads out user and message</string>
     <string name="preference_tts_message_format_key" translatable="false">tts_message_format_key</string>
     <string name="preference_tts_message_format_title">Message format mode</string>
+    <string name="preference_tts_message_filter_url_message">Reads out URLs</string>
+    <string name="preference_tts_message_filter_url_message_filter">Removes URLs from TTS</string>
```

Comment:
Maybe `Ignores links from TTS`?

Reply:
Fixed in [8ba2f1a](https://github.com/flex3r/DankChat/pull/309/commits/8ba2f1a93bf204ed3bddbffa668b5e8c446f767c)